api/src/test/kotlin/edu/wgu/osmt/elasticsearch/SearchControllerTest.kt

```diff
@@ -113,4 +121,33 @@ internal class SearchControllerTest @Autowired constructor(
         // Assert
         assertThat(result.body?.map { it.name }).contains(listOfKeywords[0].value)
     }
-}
\ No newline at end of file
+
+    @Test
+    fun similarSkillWarningsShouldFindSimilarities() {
+        richSkillRepository.createFromApi(
+            listOf(ApiSkillUpdate(
+                    "Access and Security Levels Standardization",
+                    "Standardize levels of access and security to maintain information security.",
+                    PublishStatus.Draft
+                )),
+            "admin",
+            "admin@wgu.edu"
+        )
+        val response = searchController.similarSkillWarnings(
+            arrayOf(ApiSimilaritySearch("Standardize levels of access and security to maintain information security."))
+        )
+        assertThat((response as ResponseEntity).statusCode).isEqualTo(HttpStatus.OK)
+        assertThat(response.body?.size).isEqualTo(1)
```

Comment:
Assertion in line 140 is not needed, since we just care about the content of the search, and also it would be good to add another assertion about the content like asserting the RSD statement as follows: 
`assertThat(response.body?.first()?.get(0)?.skillStatement).isEqualTo(actual[0].statement)`